model.ipynb

```diff
@@ -35,7 +35,6 @@
    "outputs": [],
```

Comment:
<gdiv class="ginger-module-highlighter-float-content" style="height: 278px;"><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025191501690995311755 gwmwi-0 ginger-module-highlighter-mistake-anim" style="position: absolute; top: 12px; width: 7px; height: 21px; left: 61px;"></gwmw><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025191507712737708407 gwmwi-0 ginger-module-highlighter-mistake-anim" style="position: absolute; top: 12px; width: 26px; height: 21px; left: 72px;"></gwmw><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025191524078038748719 gwmwi-0 ginger-module-highlighter-mistake-anim" style="position: absolute; top: 12px; width: 39px; height: 21px; left: 102px;"></gwmw></gdiv>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sarajcev/Seminar/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<gdiv class="ginger-module-highlighter-float-content" style="height: 278px;"><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025285635907150987461 gwmwi-0 ginger-module-highlighter-mistake-anim no-suggestion" style="position: absolute; top: 12px; width: 11px; height: 21px; left: 54px;"></gwmw><gwmw class="ginger-module-highlighter-mistake-type-2 gwmw-16025285635902613122701 gwmwi-0 ginger-module-highlighter-mistake-anim" style="position: absolute; top: 12px; width: 8px; height: 21px; left: 180px;"></gwmw><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025285635903882123933 gwmwi-0 ginger-module-highlighter-mistake-anim no-suggestion" style="position: absolute; top: 12px; width: 15px; height: 21px; left: 312px;"></gwmw><gwmw class="ginger-module-highlighter-mistake-type-1 gwmw-16025285635904278132077 gwmwi-0 ginger-module-highlighter-mistake-anim no-suggestion" style="position: absolute; top: 12px; width: 41px; height: 21px; left: 331px;"></gwmw></gdiv>

Reply:
<gdiv class="ginger-module-highlighter-float-content" style="height: 278px;"></gdiv>

Reply:
Ovo je još jedan način da se cikličnost vremenske serije predstavi modelu. Hammingov prozor je najsličniji kosinusnom (koji nije implementiran u pandasu).